src/iter/mod.rs

```diff
@@ -85,13 +85,27 @@ mod unzip;
 #[cfg(test)]
 mod test;
 
+/// `IntoParallelIterator` implements the conversion to a [`ParallelIterator`].
+/// By implementing `IntoParallelIterator` for a type, you define how it will
+/// transformed into an iterator. This is a parallel version of the standard
+/// library's [`std::iter::IntoIterator`] trait.
+///
+/// [`ParallelIterator`]: trait.ParallelIterator.html
+/// [`into_par_iter()`]: trait.IntoParallelIterator.html#method.into_par_iter
+/// [`std::iter::IntoIterator`]: https://doc.rust-lang.org/std/iter/trait.IntoIterator.html
 pub trait IntoParallelIterator {
     type Iter: ParallelIterator<Item = Self::Item>;
     type Item: Send;
 
     fn into_par_iter(self) -> Self::Iter;
 }
 
+/// `IntoParallelRefIterator` implements the conversion to a [`ParallelIterator`].
+/// This is a parallel version of the `iter()` method 
+/// defined by various collections.
```

Comment:
I'd like this to mirror the `IntoParallelRefMutIterator` docs, i.e. "providing shared references ...", blank line to break the paragraph after the first sentence, and mention a similar auto-implementation.